pkg/features/list.go

```diff
@@ -108,7 +108,7 @@ var (
 	KnownExploitedVulnerabilities = registerFeature("Display CISA Known Exploited Vulnerabilities (KEV) data", "ROX_CISA_KEV")
 
 	// SensitiveFileActivity enables monitoring of sensitive files.
-	SensitiveFileActivity = registerFeature("Enable sensitive file monitoring", "ROX_SENSITIVE_FILE_ACTIVITY")
+	SensitiveFileActivity = registerFeature("Enable sensitive file monitoring", "ROX_SENSITIVE_FILE_ACTIVITY", enabled)
```

Comment:
Do you intend to merge this with the feature flag enabled by default?

Reply:
Based on his commit [9c218e1](https://github.com/stackrox/stackrox/commit/9c218e109856695b3802787639cfa9b97dace33c) @Stringy seems to have enabled it temporarily, most likely while debugging the pipeline issue.